kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js

```diff
@@ -522,6 +522,11 @@ const coachStrings = createTranslator('CommonCoachStrings', {
     context:
       "Text shown on a modal pop-up window when the user clicks the 'Start Quiz' button. This explains what will happen when the user confirms the action of starting the quiz.",
   },
+  canNoLongerEditQuizNotice: {
+    message: 'You will no longer be able to edit the quiz.',
```

Comment:
This is inaccurate - it should be something like "You will no longer be able to edit the questions and sections in this quiz" - as they will still be able to edit the title and assignments.

Reply:
oof also its in the wrong file 😓